configs/openmrs_config/roles/cambodia_roles.csv

```diff
@@ -1,2 +1,4 @@
ce; Add HL7 Inbound Archive; Get HL7 Inbound Archive; Delete Orders; Get HL7 Inbound Queue; Assign System Developer Role; View RESTWS; Edit Patient Programs; Manage Location Attribute Types; Remove Problems; Edit Cohorts; Manage Concept Map Types; Edit Users; Patient Dashboard - View Patient Summary; Get Concept Attribute Types; Add HL7 Inbound Queue; Get Concept Datatypes; Get Concept Classes; Get Roles; Manage Scheduler; Get Privileges; Get Order Types; Delete Observations; Get Field Types; Get Concept Sources; Get Relationship Types; Update HL7 Inbound Archive; Update HL7 Inbound Queue; Get Identifier Types; Edit People; Delete People; Edit Patient Identifiers; View Orders; Get Forms; Get Orders; Manage Global Properties; View Patient Identifiers; Get Global Properties; Get People; Get Person Attribute Types; Get Database Changes; Get Relationships; Get Allergies; Edit Orders; Get Problems; Manage Implementation Id; Get Programs; Get Concept Reference Terms; Get Order Frequencies; View Concept Proposals; Manage Encounter Types; View Roles; Get Concept Map Types; Patient Dashboard - View Forms Section; Add HL7 Inbound Exception; Delete HL7 Inbound Queue; View People; Edit Allergies; Manage Location Tags; Manage Concept Reference Terms; Delete Diagnoses; Manage Person Attribute Types; Patient Overview - View Relationships; Patient Overview - View Programs; Patient Overview - View Problem List; Patient Overview - View Allergies; Manage Order Types; Get Encounters; Get Users; Get Locations; Manage Alerts; Get Encounter Types; Get Patients; View Identifier Types; Get Observations; Edit Observations; Get Patient Cohorts; Manage Providers; Get Patient Identifiers; Delete Users; Manage Locations; Manage Concept Name tags; Manage Metadata Mapping; Edit Conditions; Add HL7 Source; Add Users; Get Concepts; View Relationship Types; Edit Notes; Get Concept Proposals; Manage Search Index; View Relationships; Manage Relationships; Generate Batch of Identifiers; Edit User Passwords; View Person Attribute Types; View Patient Cohorts; Add Encounters; Upload Batch of Identifiers; View Field Types; View Encounter Types; Manage Identifier Types; Remove Allergies; Update HL7 Inbound Exception; View Order Types; Get Diagnoses Attribute Types; Patient Overview - View Patient Actions; Patient Dashboard - View Regimen Section; Add Cohorts; Delete Visits; Manage Order Frequencies; Edit Relationships; Manage Auto Generation Options; Manage Concepts; Manage Concept Datatypes; Add People; Manage Encounter Roles; View Privileges; Delete Patients; Delete Concept Proposals; Manage Visit Types; Delete Conditions; View Concept Datatypes; Edit Patients; Delete Notes; Edit Encounters; View Concepts; Manage Visit Attribute Types; Form Entry; View Global Properties; Get Care Settings; Manage Concept Sources; Manage Order Sets; Get Order Set Attribute Types; Manage Relationship Types; Get Diagnoses; Edit Visits; View Users; Share Metadata; View Forms; Get Forms;View Patient Summary,
```

Comment:
Could we make this title case? (Sorry I know this is not title case in the ticket description, but I feel it will look better).